content/en/seachat/seachat-manual/02-create-agent/04-agent-information.md

```diff
@@ -0,0 +1,114 @@
+
+You can disable this feature by unchecking the box such that your user will not be able to make a request for a live agent during the conversation, while the live agent can still take over the conversation from the **Conversations** dashboard. You can visit our [Live Agent Tutorial](/seachat/seachat-manual/02-create-agent/04-agent-information/) for more detailed explanation of the feature. 
+
+### Interact with AI Agent
+You can immediately interact with the AI agent that you have just built. In **Agent URL**, you can click the URL to interact with your AI agent in a standalone window. 
+
+Or if you wish to integrate your AI agent into your website, you can also access your API key in the **API Key** section. Please check out the [Webpage Integration](/seachat/seachat-manual/04-channels/08-install-to-webpage/) for more information on how to programmatically integrate your AI agent into your website.
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/seachat/en/agent-information/additional-options.png" target="_blank">
+    <img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/agent-information/additional-options.png" alt="image that displays the additional options in Basic Settings">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Additional Options in Basic Settings</p>
+</div>
+
+## Advanced Settings
+
+Different from the basic settings, advanced settings allow you to customize your AI agent with more advanced features that SeaChat has curated for you. Inside **Advanced Settings** under **Agent Information**, the following advanced features are available:
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/seachat/en/agent-information/advanced-settings-dashboard.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/agent-information/advanced-settings-dashboard.png" alt="image that displays the additional options in Basic Settings">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Advanced Settings Dashboard</p>
+</div>
+
+### Agent Memory
+Agent memory fields are used to extract information from a conversation and store it for future reference. Memory is a powerful yet complex feature. We recommend you refer to the [Agent Memory](/seachat/seachat-manual/02-create-agent/03-agent-memory/) tutorial to learn more about how to set up memory fields for your AI agent.
```

Comment:
Add a few reasons why memory is a good feature. You can build customer profiles, prospect/qualify customers before further engagements etc